internal/repository/gorm/helpers_test.go

```diff
@@ -13,12 +13,15 @@ import (
 	ints "github.com/porter-dev/porter/internal/models/integrations"
 	"github.com/porter-dev/porter/internal/repository"
 	"github.com/porter-dev/porter/internal/repository/gorm"
+
+	_gorm "gorm.io/gorm"
 )
 
 type tester struct {
 	repo           repository.Repository
 	key            *[32]byte
 	dbFileName     string
+	DB             *_gorm.DB
```

Comment:
```suggestion
	db             *_gorm.DB
```

This shouldn't be exported